docs/releasing.md

```diff
@@ -25,35 +43,104 @@ Release manager follows the below _Release Process_.
 
 
 ## Release Process
 
+Note: You'll wanna be on a Linux machine, since the Lightrider step will require that.
+
+### On the scheduled release date
+
+Before starting, you should announce to the LH eng channel that you are releasing,
+and that no new PRs should be merged until you are done.
+
+```sh
+# Run the tests.
+bash ./lighthouse-core/scripts/release/test.sh
+# Change into the pristine folder.
+cd ../lighthouse-pristine
+```
+
+Confirm DevTools integration will work:
+```sh
+# You should have Chromium already checked out at ~/chromium/src
+# See: https://www.chromium.org/developers/how-tos/get-the-code
+
+# Roll to Chromium folder.
+yarn devtools
+
+# Checkout latest Chromium code.
+cd ~/chromium/src
+git pull
+git new-branch lh-roll-x.x.x
+gclient sync
+autoninja -C out/Release chrome blink_tests
+
+# Run tests and rebase.
+yarn --cwd ~/chromium/src/third_party/blink/renderer/devtools test 'http/tests/devtools/audits/*.js' --reset-results
+# Verify the changes are expected.
+git diff
+
+# Verify that the Audits panel still works. Consider the new features that have been added.
+# If anything is wrong, stop releasing, investigate, and prioritize landing the PR.
+
+# For bonus points, add some tests covering new features. Either a new test, or an extra
+# assertion in an existing test.
+
+git cl upload --bypass-hooks
+# Go to Gerrit, run CQ dry run, ensure the tests all pass.
+```
+
+Confirm Lightrider integration will work:
+```sh
+# See the internal README for updating Lighthouse.
+
+# Test things out locally, if happy, deploy to canary and see how the graphs react. 20 minutes should be enough time.
+
+# Do the stuff in "Test LR changes in Canary".
+# https://g3doc.corp.google.com/chrome/headless/lightrider/README.md?cl=head#test-lr-changes-in-canary
+
+# Verify that Lightrider works properly, and is generating reports fully. Consider the new features that have been added.
+# Note: if the changes include proto changes make sure that the API has those new fields.
+# If anything is wrong, stop releasing, investigate, and prioritize landing the PR.
+
+# For bonus points, add some tests covering new features. Either a new test, or an extra
+# assertion in an existing test.
+```
+
+Now that the integrations are confirmed to work, go back to `lighthouse` folder.
+
 ```sh
-# Run the tests
+# Run the tests again.
+# Note: you can skip this if you didn't need to land any changes.
 bash ./lighthouse-core/scripts/release/test.sh
+
 # Prepare the commit, replace x.x.x with the desired version
 bash ./lighthouse-core/scripts/release/prepare-commit.sh x.x.x
 
 # Open the PR and await merge...
-echo "It's been merged! 🎉"
+echo "It's been merged! 🎉"
 
-# Run the tests again :)
+# One last test (this script uses origin/master, so we also get the commit with the new changelog - that commit should be HEAD).
```

Comment:
should this step be run in pristine?

Reply:
The release scripts cd to pristine. You always run them  from a development checkout.